content/manuals/scout/explore/metrics-exporter.md

```diff
@@ -37,12 +37,13 @@ The metrics endpoint exposes the following metrics:
 ## Creating an access token
 
 To export metrics from your organization, first make sure your organization is enrolled in Docker Scout.
-Then, create a Personal Access Token (PAT) - a secret token that allows the exporter to authenticate with the Docker Scout API.
+Then, create an Organization Access Token (OAT) - a secret token that allows the exporter to authenticate the organization with the Docker Scout API.
 
-The PAT does not require any specific permissions, but it must be created by a user who is an owner of the Docker organization.
-To create a PAT, follow the steps in [Create an access token](/security/for-developers/access-tokens/#create-an-access-token).
+The access token does not require any specific permissions, but it must be an organization access token created for the Docker organization in question,
+or a personal access token created by a user who is an owner of the Docker organization.
+For instructions on how to create an access token, follow the steps in [Create an access token](/security/for-developers/access-tokens/#create-an-access-token).
```

Comment:
This section confuses me. In the first paragraph, we say explicitly "Create an Organization Access Token", and in this paragraph it seems like I could use either an OAT or PAT as my access token. It might be helpful to define "Access token" as a word, and that OATs or PATs are two types of access tokens the user can choose from (with the requirements for each as outlined here)

Reply:
fair, I've reworded the section a bit, hope it's clearer.